src/config/firebase.js

```diff
@@ -15,8 +40,12 @@ if (isBrowser && !firebase.apps.length) {
   firebase.initializeApp(config);
 
   try {
-    firebase.analytics();
+    disableGoogleAnalyticsAdSignals();
+    const analytics = firebase.analytics();
+    analytics.setAnalyticsCollectionEnabled(shouldEnableAnalytics());
   } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('Failed to initialise analytics', error);
```

Comment:
Why do we need this console as well?

Reply:
At the moment, errors here are only sent to Rollbar, which I don't enable locally, so it's useful for local development (e.g. in case of missing Firebase token, etc.)